src/components/VideoPlayer/BaseVideoPlayer.tsx

```diff
@@ -310,7 +349,10 @@ function BaseVideoPlayer({
                                     if (isFullScreenRef.current) {
                                         return;
                                     }
-                                    togglePlayCurrentVideo();
+                                    if (!isSmallScreenWidth) {
+                                        togglePlayCurrentVideo();
+                                    }
+                                    showControl();
```

Comment:
NAB but can we prevent calling `showControl` for large screen devices? Maybe via an early return after calling `togglePlayCurrentVideo`. It will also keep possible future regressions at bay.